tests/cypress/integration/api/graphqlOrdering.spec.ts

```diff
@@ -0,0 +1,66 @@
+import gql from 'graphql-tag'
+
+describe('Test graphql ordering', () => {
+    before('create a list with 3 children', () => {
+        cy.apollo({
+            mutation: gql`
+                mutation {
+                    jcr(workspace: EDIT) {
+                        addNode(parentPathOrId: "/", name: "testList", primaryNodeType: "jnt:contentList") {
+                            subnode1: addChild(name: "Hello", primaryNodeType: "jnt:bigText") {
+                                uuid
+                            }
+                            subnode2: addChild(name: "Bonjour", primaryNodeType: "jnt:press") {
+                                uuid
+                            }
+                            subnode3: addChild(name: "Hola", primaryNodeType: "jnt:linkList") {
+                                uuid
+                            }
+                        }
+                    }
+                }
+            `,
+        })
+    })
+
+    after('Delete', function () {
+        cy.apollo({
+            mutationFile: 'jcr/deleteNode.graphql',
+            variables: {
+                pathOrId: '/testList',
+            },
+        })
+    })
+
+    it('Should order by nodeType', () => {
+        cy.apollo({
+            queryFile: 'ordering.graphql',
```

Comment:
The filename is `queryFile` does not match the name of the file you want to load.